src/main/scala/org/broadinstitute/dsde/rawls/model/ExecutionModel.scala

```diff
@@ -0,0 +1,79 @@
+package org.broadinstitute.dsde.rawls.model
+
+import scala.annotation.meta.field
+import spray.json._
+import com.wordnik.swagger.annotations.{ApiModel, ApiModelProperty}
+import org.joda.time.DateTime
+import org.broadinstitute.dsde.rawls.VertexProperty
+
+/**
+ * @author tsharpe
+ */
+
+// this is what a client sends to have us create a submission
+
+// Request for a submission
+@ApiModel(value = "Submission")
+case class Submission(
+  @(ApiModelProperty@field)(required = true, value = "Namespace of the method configuration to execute.")
+  methodConfigurationNamespace: String,
+  @(ApiModelProperty@field)(required = true, value = "Name of the method configuration to execute.")
+  methodConfigurationName: String,
+  @(ApiModelProperty@field)(required = true, value = "Type of root entity for expression.")
+  entityType: String,
+  @(ApiModelProperty@field)(required = true, value = "Name of root entity for expression.")
+  entityName: String,
+  @(ApiModelProperty@field)(required = false, value = "Expression that resolves to one or more entities matching the entity type in the method configuration.")
+  expression: Option[String]
+)
+
+// Cromwell's response to job submission (not decorated with annotations because it's not part of our API)
+case class JobStatus(
+  id: String,
+  status: String
+)
+
+// Status of a workflow
+@ApiModel(value = "WorkflowStatus")
+case class WorkflowStatus(
```

Comment:
can we rename this Workflow? I want to make a WorkflowStatus enum that contains the actual status values and having this be called WorkflowStatus makes it awkward.